english/modern-tech

```diff
@@ -0,0 +1,2 @@
+If you would like to learn more about today's modern tech please Click [here](http://phys.org/technology-news/software/)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.